rebar.config

```diff
@@ -28,7 +28,8 @@
 
 {erl_opts, [
   debug_info,
-  {platform_define, "19|^2", 'UDP_HEADER'}
+  {platform_define, "^18", 'UDP_HEADER_1'},
+  {platform_define, "^19|^20|^21|^22.0", 'UDP_HEADER_2'}
```

Comment:
Can the platform for OTP 22 be upped to `^22` to include all versions of OTP 22? I use this library, and would like to use it on OTP 22.1. I was going to open an issue but I hope just a comment here is okay.

Reply:
The UDP header changed in OTP 22.1, this is why I explicitly check for 22.0. All release post 22.0 (including 22.1) should work until they change the UDP headers again.

Reply:
Thanks for the response and for merging this in!